free-1.md

```diff
@@ -10,41 +10,120 @@ _What need is met by your idea? Who are you helping? What is that person's
+animations. This DSL meets the need of inspiring creativity through code! It
+allows users to create animated art work through simple programming. Animation 
+can be a very tedious and intimidating process for those who don't have strong 
+drawing skills or experience with the Adobe Creative Suite. In other words, 
+animation seems out of reach for those who don't really have an art background 
+and are just casually interested in animation. This DSL allows for users to 
+create cool animations with very little to no background in drawing or coding! 
+I imagine it would be a great learning tool for kids, inspiring interest in 
+the fields of both Computer Science and Art.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would allow the user to more easily create looping animations. For 
+example, it's likely easier to define and add animations to large number of 
+shapes through code than through manually drawing them or adding new shapes
+through some sort of GUI. It addresses the desire to create interesting 
+looping patterns and movements with a relatively little investment in time.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I'm excited by this idea because I'm personally interested in learning 
+about animation but have very little experience with it! I created very 
+minor animations using Unity this summer, but found it to be a rather 
+painstaking process and required a great deal of time and patience to 
+learn. I reflected on this experience and thought about how a DSL could
+be used to facilitate this process. I thought back to our first assignment 
+with ContextFree, and I thought it would be cool to apply principles of this
+DSL in creating a new DSL that allows for animations.
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+Digital art with looping animations
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
+like? Why is this the right way to interact with the problem domain?_
+
+I want to keep to commands as simple as possible, so there would be a small
+set of defined shapes, which the user would instantiate as objects. The 
+properties of the object could be modified (i.e. color and size), and the
+objects could be easily modified. There would also be a set of animation
+methods, which could allow shapes to float across the screen, rotate in place,
+and gradually change size/color. The user would call methods on the objects 
+and they could specifiy the intervals at which the methods would be called.
+I imagine the interface looking similar to ContextFree, where the user writes 
+code in one part of the window, and the result of compiled code appears next 
+to it. This would be a good way for the user to immediately see the results
```

Comment:
A really easy way to do this might be to have your code compile down to HTML/CSS/SVG and use webkit (or similar) to draw and animate them.